### ISSUE TITLE
Show latests comments on index.

### ISSUE THREAD

**Issue by Sama34** [medbenji/MyProfile#54]

**Comment by dragonexpert** [comment]
I prefer this kind of feature to be a moderator one.
1) Moderators need to be able to police them.
2) Members aren't generally going to need to see what is on other people's profiles on the index page.  Threads are different because they are able to reply to them.
3) Permissions would have to be taken into consideration if members were able to view this.

**Comment by Sama34** [comment]
I don't think it is hard to implement this. Also note that the comments feature could be used as a status one.

I have also been thinking about showing my comments to other users profiles in my own comment list. A la Facebook. I would like to think of MyProfile as a full profile mod pack.
